moz-webgpu-cts/src/main.rs

```diff
@@ -125,6 +190,474 @@ fn run(cli: Cli) -> ExitCode {
     }
 
     match subcommand {
+        Subcommand::ProcessReports {
+            report_globs,
+            report_paths,
+            preset,
+        } => {
+            let report_globs = {
+                let mut found_glob_parse_err = false;
+                // TODO: I think these can just be args. directly?
+                let globs = report_globs
+                    .into_iter()
+                    .filter_map(|glob| match Glob::diagnosed(&glob) {
+                        Ok((glob, _diagnostics)) => Some(glob.into_owned().partition()),
+                        Err(diagnostics) => {
+                            found_glob_parse_err = true;
+                            let error_reports = diagnostics
+                                .into_iter()
+                                .filter(|diag| {
+                                    // N.B.: There should be at least one of these!
+                                    diag.severity()
+                                        .map_or(true, |sev| sev == miette::Severity::Error)
+                                })
+                                .map(Report::new_boxed);
+                            for report in error_reports {
+                                eprintln!("{report:?}");
+                            }
+                            todo!("render glob parse diagnostics in hard error")
```

Comment:
- [ ] Do it.

Reply:
I think current behavior is fine? Will let future experiences inform what to do here. Fixed up with 09caea1d7b2d0b0bf0e4ab21c11835764d20b7b1.